docs/user/forms.md

```diff
@@ -94,14 +93,14 @@ for page in reader.pages:
 
-However, it's also important to note that the two lists do not *always* refer to the same underlying PDF object. For example, if the form contains radio buttons, you will find that `reader.get_fields()` will get the parent object (the group of radio buttons) whereas `page.annotations` will return all the child objects (the individual radio buttons).
+However, it is also important to note that the two lists do not *always* refer to the same underlying PDF object. For example, if the form contains radio buttons, you will find that `reader.get_fields()` will get the parent object (the group of radio buttons) whereas `page.annotations` will return all the child objects (the individual radio buttons).
 
-__Caution: Remember that fields are not stored in pages: If you use  `add_page()` the field structure is not copied. It is recommended to use `.append()` with the proper parameters instead.__
+__Caution: Remember that fields are not stored in pages; if you use  `add_page()` the field structure is not copied. It is recommended to use `.append()` with the proper parameters instead.__
 
-In case of missing _field_ objects in `/Fields`, `writer.reattach_fields()` will parse page(s) annotations and will reattach them. This fix can not guess intermediate fields and will not report fields using the same _name_.
+In case of missing _field_ objects in `/Fields`, `writer.reattach_fields()` will parse page(s) annotations and will reattach them. This fix cannot guess intermediate fields and will not report fields using the same _name_.
 
 ## Identify pages where fields are used
 
-On order to ease locating page fields you can use `page.get_pages_using_field`. This methods accepts a field object, id est a *PdfObject* that represents a field (as are extracted from `_root_object["/AcroForm"]["/Fields"]`. The method returns a list of pages, because a field can have multiple widgets as mentioned previously (e.g. radio buttons or text displayed on multiple pages).
+In order to ease locating page fields you can use `page.get_pages_using_field`. This methods accepts a field object, that is a *PdfObject* that represents a field (as extracted from `_root_object["/AcroForm"]["/Fields"]`). The method returns a list of pages, because a field can have multiple widgets as mentioned previously (e.g. radio buttons or text displayed on multiple pages).
```

Comment:
```suggestion
In order to ease locating page fields you can use `page.get_pages_using_field`. This methods accepts a field object, a *PdfObject* that represents a field (as extracted from `_root_object["/AcroForm"]["/Fields"]`). The method returns a list of pages, because a field can have multiple widgets as mentioned previously (e.g. radio buttons or text displayed on multiple pages).
```

Reply:
I think removing the "that is" improves this. Also change "This methods" to "This method" at the start of the second sentence.

Reply:
Is this correct: page.get_pages_using_field? It is not documented in https://pypdf.readthedocs.io/en/latest/modules/PageObject.html.

Reply:
It is part of the reader and writer and indeed named `get_pages_showing_field`.

Reply:
@stefan6419846 get_pages_using_field is only used in this one place. I have updated it to get_pages_showing_field.